integration-cli/docker_cli_netmode_test.go

```diff
@@ -14,152 +12,80 @@ import (
 // the command executed in a container did really run PS correctly.
 const stringCheckPS = "PID   USER"
 
-// checkContains is a helper function that validates a command output did
-// contain what was expected.
-func checkContains(expected string, out string, c *check.C) {
-	if !strings.Contains(out, expected) {
-		c.Fatalf("Expected '%s', got '%s'", expected, out)
-	}
+// DockerCmdWithFail executes a docker command that is supposed to fail and returns
+// the output, the exit code. If the command returns an Nil error, it will fail and
+// stop the tests.
+func dockerCmdWithFail(c *check.C, args ...string) (string, int) {
```

Comment:
why not exposing this in docker_utils? This pattern is often used elsewhere also